### PR TITLE
Update `bump_frontend` command to my workflow

### PR DESCRIPTION
**Pull request by bramkragten**
This makes it easier for me, not sure if it fits everybody 😄

### CODE REVIEW

File: hassrelease/commands.py [hunk 1]
```diff
@@ -230,15 +230,43 @@ def credits(simul_requests, no_cache, quiet):
...
+    branch_name = f"bump-frontend-{frontend}"
+
+    if create_branch:
+        git.create_branch(repo_core.PATH, branch_name)
+
+    repo_core.update_frontend_version(frontend)
+    repo_core.gen_requirements_all()
+    repo_core.commit_all(f"Update frontend to {frontend}")
+
+    if create_branch:
```
Comment (balloob) [inline hassrelease/commands.py:267]: Bonus: print the PR url
```suggestion
    if create_branch:
        print(
            f"PR link: https://github.com/home-assistant/home-assistant/compare/dev...{branch_name}?expand=1&title=Update%20frontend%20to%20{frontend}&body=https://github.com/home-assistant/frontend/releases/tag/{frontend}"
        )
```

File: hassrelease/git.py [hunk 2]
```diff
@@ -83,3 +83,55 @@ def is_dirty(repo):
         ).stdout
         != b""
     )
+
+
+def is_main(repo):
+    """Test if current repo is the main repo."""
```
Comment (balloob) [inline hassrelease/git.py:89]:
```suggestion
    """Test if current branch is the main branch."""
```

File: hassrelease/git.py [hunk 3]
```diff
@@ -83,3 +83,55 @@ def is_dirty(repo):
...
+        subprocess.run(
+            "git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@'",
+            capture_output=True,
+            shell=True,
+            cwd=repo,
+        )
+        .stdout.decode("UTF-8")
+        .replace("\n", "")
+    )
+
```
Comment (balloob) [inline hassrelease/git.py:131]:
```suggestion

    assert branch != default

```

File: hassrelease/commands.py (hunk 1 above)
Comment (bramkragten) [reply hassrelease/commands.py:267]: It is already printed by github

File: hassrelease/git.py [hunk 4]
```diff
@@ -83,3 +83,56 @@ def is_dirty(repo):
...
+    assert branch != default
```
Comment (bramkragten) [inline hassrelease/git.py:132]:
```suggestion
    assert branch != default_branch
```

**Review by balloob — APPROVED** [review]

**Pull request merged**